### PR TITLE
Add human-doc for msi packet

### PR DESCRIPTION
**Pull request by NamsooCho**

### CODE REVIEW

File: src/messenger_packets.md [hunk 1]
```diff
@@ -146,3 +146,115 @@ Length    | Content
...
+`1`       | `0x45`
+`0..255`  | payload
+
+Though protocol defines `size` field, actually every commands's size of `value` is 1 byte.
+So, `size` is for the future extension.
+
+This packet structure can permit for a node to send multiple commands in one packet.
+For example a node can send packet to a friend to start call with capabilities and errors.
+The node receiving this packet also can do multiple action on commands in a packet.
+Because `payload` size is 255 bytes long, we can send maximum 255 / 3 = 85 commands in a packet.
```
Comment (kpp) [inline src/messenger_packets.md]: > 255 / 3 = 85 commands in a packet

3 bytes per a command?

File: src/messenger_packets.md [hunk 2]
```diff
@@ -146,3 +146,115 @@ Length    | Content
...
+```
+
+Protocol: |kind [1 byte]| |size [1 byte]| |value [$size bytes]| |...{repeat}| |0 {end byte}|
+
+Serialized form:
+
+Length    | Content
+--------- | ------
+`1`       | `0x45`
+`0..255`  | payload
```
Comment (kpp) [inline src/messenger_packets.md:183]: So what's inside the payload?

File: src/messenger_packets.md (hunk 1 above)
Comment (NamsooCho) [reply src/messenger_packets.md]: Yes.

>Though protocol defines size field, actually every commands's size of value is 1 byte. So, size is for the future extension.

File: src/messenger_packets.md (hunk 2 above)
Comment (NamsooCho) [reply src/messenger_packets.md:183]: done

Comment (kpp) [reply src/messenger_packets.md:183]: Still don't get it

**Pull request merged**